sentry.properties

```diff
@@ -0,0 +1,4 @@
+defaults.url=https://sentry.io/
+defaults.org=fewlines
+defaults.project=connect-account
+cli.executable=../../../.npm/_npx/47675/lib/node_modules/@sentry/wizard/node_modules/@sentry/cli/bin/sentry-cli
```

Comment:
Does this work for CI too?

Reply:
What do you mean by this?
Tbh, I don't really see the point of this. The wizard is just here to (optionally) automate the process of implementation. I don't see why we would need this afterward.

Reply:
> What do you mean by this?


I mean that `../../../.npm` is not in this repo I think ^^

Reply:
I had a hard time finding some info regarding this file, but let me double check. I'm pretty sure I can get ride of this entry but I want to be sure this is not used internally (which would be super odd/bad, but who knows).

Reply:
https://github.com/getsentry/examples/blob/master/react-native/sentry.properties
Ok, I'm removing it 👍.